workshop/content/english/40-dotnet/_index.md

```diff
@@ -0,0 +1,9 @@
++++
```

Comment:
Please ensure this file is up to date - there should be a warning talking about cost